src/main/java/nl/beehivejava/scriptj/PluginManager.java

```diff
@@ -0,0 +1,14 @@
+package nl.beehivejava.scriptj;
+
+/**
+ * @author Lesley
+ */
+public interface PluginManager {
+
+    Plugin load(String location);
+
+    void addEnvironment(PluginEnvironment environment);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)